<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="CacheInfoWidget" name="cacheInfoWidget" native="true">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>460</width>
      <height>440</height>
     </rect>
    </property>
   </widget>
   <widget class="TabsWidget" name="tabsWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionUpdate"/>
    <addaction name="actionDebug_mode"/>
    <addaction name="actionDebug_mode_off"/>
   </widget>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionUpdate">
   <property name="text">
    <string>Update</string>
   </property>
  </action>
  <action name="actionDebug_mode">
   <property name="text">
    <string>Debug mode</string>
   </property>
  </action>
  <action name="actionDebug_mode_off">
   <property name="text">
    <string>Debug mode off</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CacheInfoWidget</class>
   <extends>QWidget</extends>
   <header>CacheInfoWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabsWidget</class>
   <extends>QWidget</extends>
   <header location="global">TabsWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
